db/migrate/20181001152012_rename_solution_mentoring_column.rb

```diff
@@ -0,0 +1,11 @@
+class RenameSolutionMentoringColumn < ActiveRecord::Migration[5.2]
+  def up
+    add_column :solutions, :mentoring_requested_at, :datetime, null: true
+    Solution.where(independent_mode: false).
```

Comment:
Does this mean that all solutions in mentored mode will have mentoring requested? Including side exercises?